scripts/qa/gofmt.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# this script checks whether all files (except vendorred and generated files)
+# are properly formatted and simplified with gofmt.
+
+# find the dir we exist within...
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+# and cd into root project dir
+cd ${DIR}/../..
+
+out=$(gofmt -d -s $(find . -name '*.go' | grep -v vendor | grep -v _gen.go))
+if [ "$out" != "" ]; then
+	echo "$out"
+	echo
+	echo "You might want to run something like 'find . -name '*.go' | xargs gofmt -w -s'"
```

Comment:
```suggestion
	echo "You might want to run something like 'find . -name '*.go' -not -path './vendor/*' | xargs gofmt -w -s'"
```

I know it's not hard to revert the changes to vendor in git, but this is easier. We should change it to this everywhere else in our code as well.